nixos/modules/services/misc/jellyfin.nix

```
@@ -1,33 +1,70 @@
 { config, pkgs, lib, ... }:
 
-with lib;
-
 let
+  inherit (lib.modules) mkIf;
+  inherit (lib.meta) getExe maintainers;
+  inherit (lib.options) mkEnableOption mkOption mkPackageOption mdDoc;
+  inherit (lib.types) str path bool;
+
   cfg = config.services.jellyfin;
 in
 {
   options = {
     services.jellyfin = {
-      enable = mkEnableOption (lib.mdDoc "Jellyfin Media Server");
+      enable = mkEnableOption (mdDoc "Jellyfin Media Server");
 
       user = mkOption {
-        type = types.str;
+        type = str;
         default = "jellyfin";
-        description = lib.mdDoc "User account under which Jellyfin runs.";
+        description = mdDoc "User account under which Jellyfin runs.";
       };
 
       package = mkPackageOption pkgs "jellyfin" { };
 
       group = mkOption {
-        type = types.str;
+        type = str;
         default = "jellyfin";
-        description = lib.mdDoc "Group under which jellyfin runs.";
+        description = mdDoc "Group under which jellyfin runs.";
+      };
+
+      dataDir = mkOption {
+        type = path;
+        default = "/var/lib/jellyfin";
+        description = mdDoc ''
+          base data directory, see https://jellyfin.org/docs/general/administration/configuration/#data-directory
+        '';
+      };
+
+      configDir = mkOption {
+        type = path;
+        default = "${cfg.dataDir}/config";
+        defaultText = "\${cfg.dataDir}/config";
+        description = mdDoc ''
+          directory containing the server configuration files.
```

```suggestion
          Directory containing the server configuration files.
```
Sentences start with a capital letter. Same applies to the next descriptions.